scripts/dune

```diff
@@ -0,0 +1,9 @@
+(install
+ (section bin)
+ (package opam-coq-archive)
+ (files (archive2web.exe as archive2web)))
```

Comment:
I don't get the purpose of this `install` stanza. If you give a `public_name` and a `package` to the `executable` stanza below, it should be enough (see https://dune.readthedocs.io/en/latest/dune-files.html#executable).

Reply:
Fixed, thanks.